roles/chrony/templates/chrony.conf

```diff
@@ -24,8 +24,10 @@ hwtimestamp {{ chrony_hwtimestamp_interfaces }}
 minsources {{ chrony_num_minsources }}
 {%- endif %}
 
-{% if chrony_ntp_client is defined -%}
-allow {{ chrony_ntp_client }}
+{% if chrony_ntp_clients is defined -%}
```

Comment:
If we're going to rename `chrony_ntp_client` to a list, we need to update the `defaults/main.yml` as well as README.md. We should also check to be sure no one using the old value will be negatively impacted.

As an alternative we can keep both options and just deprecate the singular one.

Reply:
I agree, I'll make changes accordingly tomorrow.